lib/tasks/edition.rake

```diff
@@ -0,0 +1,12 @@
+namespace :edition do
+  desc 'Check if any deleted edition has undeleted attachments, and delete them'
+  task delete_edition_files: :environment do
+    Edition.where(state: :deleted).find_each do |edition|
+      edition.attachments.each do |attachment|
+        next if attachment.deleted?
+        puts "deleted edition #{id} has undeleted attachment with url #{attachment.url}"
```

Comment:
If we're outputting the attachment URL here, should be make https://github.com/alphagov/asset-manager/pull/560 accept the URL - or can we extract the UUID from the URL here?

Reply:
I think I'll just change the task in that PR to take the legacy URL, rather than the UUID.  I don't think we can get the UUID in whitehall without talking to asset-manager, which would be a lot of overhead.